components/shared_code/src/shared_data_model/sources/sonarqube.py

```diff
@@ -194,81 +171,109 @@ def violation_entity_attributes(
                 "Web:LongJavaScriptCheck",
                 "abap:S104",
                 "c:S104",
-                "c:S138",
                 "c:S1151",
+                "c:S138",
                 "cpp:S104",
-                "cpp:S138",
                 "cpp:S1151",
                 "cpp:S1188",
+                "cpp:S138",
```

Comment:
No `dart:S138`?

Reply:
I've only checked additional rules for the languages with profiles in SonarQube, not sure what the spread would be for other languages. Some languages have different naming for rules :( But there does not seem to be a `dart:S138`: https://next.sonarqube.com/sonarqube/coding_rules?q=dart:S138